tree/tree/inc/TBranch.h

```diff
@@ -115,6 +122,7 @@ class TBranch : public TNamed , public TAttFill {
    TBuffer    *fEntryBuffer;      ///<! Buffer used to directly pass the content without streaming
    TBuffer    *fTransientBuffer;  ///<! Pointer to the current transient buffer.
    TList      *fBrowsables;       ///<! List of TVirtualBranchBrowsables used for Browse()
+   std::unique_ptr<ROOT::Experimental::Internal::TBulkBranchRead> fBulk; ///<! Helper for performing bulk IO
```

Comment:
To reduce the code length, it might be worth have a local using/alias statement for TBulkBranchRead